config/fineractdev-cleanup.xml

```diff
@@ -1,4 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
```

Comment:
Not sure why this is suddenly needed here, and what it has to do with FINERACT-1189, but doesn't really hurt much either, not a reason to request removing it. I do expect that likely will be a small PITA for anyone ever making changes to this file again - because it's exported from Eclipse, not manually edited, and when Eclipse rewrites it, it will most likely loose this ASL header again, so whoever makes such changes will have to manually put it back (if we're enforcing this now, else we'll just loose it again sooner or later).